examples/init.d/node_exporter

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+RETVAL=0
+PROG="node_exporter"
+EXEC="/etc/node_exporter/node_exporter"
+LOCKFILE="/var/lock/subsys/$PROG"
+OPTIONS="-web.listen-address=:9201"
+
+# Source function library.
+if [ -f /etc/rc.d/init.d/functions ]; then
+  . /etc/rc.d/init.d/functions
+else
+  echo "/etc/rc.d/init.d/functions is not exists"
+  exit 0
+fi
+
+start() {
+  if [ -f $LOCKFILE ]
+  then
+    echo $"$PROG is already running!"
```

Comment:
Why `$".."`? This will interprete escape sequences which isn't needed here, right? Or does the `/etc/rc.d/init.d/functions` do something to these vars to colorize etc?

Reply:
Yes it is not needed. I removed it.